backend/embassyd.service

```diff
@@ -9,6 +9,9 @@ Environment=RUST_LOG=embassyd=debug,embassy=debug
 ExecStart=/usr/local/bin/embassyd
 Restart=always
 RestartSec=3
+ManagedOOMPreference=avoid
```

Comment:
So, fwiw embassyd was oomkilled with this option defined. Do any other changes in this PR fix this?